README.md

```diff
@@ -0,0 +1,134 @@
+This is a client implementation of the Apache Camel Language Server Protocol for Emacs
+
+# Camel LSP client for Emacs
+
+Code completion for Camel LS with XML LS
+
+![Demo](images/camell.gif)
+
+NOTE: Completion is with Ctrl+Alt+i 
+
+## Prerequisites
```

Comment:
Instead of prerequisite I would call it more like "How to configure Emacs". In fact maybe another chapter for prerequisite might be needed: java 11+ is required to be present on the system path